packages/@webex/plugin-meetings/src/constants.ts

```diff
@@ -1331,3 +1324,24 @@ export const MEETING_PERMISSION_TOKEN_REFRESH_REASON = 'ttl-join';
 
 // constant for named media group type
 export const NAMED_MEDIA_GROUP_TYPE_AUDIO = 1;
+
+export enum MeetingType {
```

Comment:
see this page about enums:
https://confluence-eng-gpk2.cisco.com/conf/x/uluyEw
and consider following the proposed approach there, you can see examples doing that in SDK: NETWORK_STATUS or IP_VERSION that follow that convention

Reply:
changed the enum implementation and renamed it to DESTINATION_TYPE